tin-drink/albert.js

```diff
@@ -0,0 +1,6 @@
+function main(){
+    var zyz=5;
```

Comment:
Expected indentation of 2 spaces but found 4.
Unexpected var, use let or const instead.
'zyz' is assigned a value but never used.